hub/powertoys/peek.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: PowerToys Peek utility for Windows
 description: A system-wide utility for Windows to quickly preview file content.
-ms.date: 11/19/2024
-ms.topic: concept-article
+ms.date: 08/03/2023
```

Comment:
The ms.date should be updated to something more current.

Reply:
Ah. Sorry about this, the ms.topic and Customer Intent changes. This was because I edited the page as it was back in October when the original PR was raised. I will restore them in a new commit.